cmd/zq/zq.go

```diff
@@ -92,8 +92,8 @@ type Command struct {
 func New(f *flag.FlagSet) (charm.Command, error) {
 	cwd, _ := os.Getwd()
 	c := &Command{dt: resolver.NewTable()}
-	f.StringVar(&c.ifmt, "i", "auto", "format of input data [auto,bzson,ndjson,zeek,zjson,zson]")
-	f.StringVar(&c.ofmt, "f", "zson", "format for output data [text,table,zeek,ndjson,bzson,zson]")
+	f.StringVar(&c.ifmt, "i", "auto", "format of input data [auto,bzng,ndjson,zeek,zjson,zng]")
+	f.StringVar(&c.ofmt, "f", "zng", "format for output data [text,table,zeek,ndjson,bzng,zng]")
```

Comment:
Nit: Sort the list of output formats for readability.

Reply:
+1